Sources/OktaIdx/Extensions/IDXClientError+Extensions.swift

```diff
@@ -158,10 +173,28 @@ extension IDXClientError: CustomNSError {
             return [
                 "name": name
             ]
+        case .missingRemediationOption(name: let name):
+            return [
+                "name": name
+            ]
         case .unknownRemediationOption(name: let name):
             return [
                 "name": name
             ]
+        case .oauthError(summary: let summary, code: let code, errorId: let errorId):
```

Comment:
nit: what do you think about `let .oauthError(summary code, errorId)`?

Reply:
This is in a switch block, so I don't believe the `if case let` syntax will work here.